common/src/main/java/com/wynntils/features/chat/DeathCoordinatesFeature.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.chat;
+
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.features.Feature;
+import com.wynntils.models.characterstate.event.CharacterDeathEvent;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.ClickEvent;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.HoverEvent;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.CHAT)
+public class DeathCoordinatesFeature extends Feature {
+
+    @SubscribeEvent
+    public void onChatReceived(CharacterDeathEvent e) {
+
+        MutableComponent deathMessage = Component.translatable("feature.wynntils.deathCoordinates.diedAt")
+                .withStyle(ChatFormatting.DARK_RED);
+        MutableComponent coordMessage = Component.translatable(
```

Comment:
We have some utility function to create a component with a build-in on click... lemme check...

Reply:
I thought we had, but I can't find it now. Nevertheless, the coordinate format is quite non-standard; we use `[123, 4, 567]` everywhere else. And that does not need to be a i18n string.

Reply:
Found it! In ChatCoordinatesFeature.createLocationComponent. You can move it to ComponentUtils and re-use it instead, that'd be much better than to reimplement it.